### PR TITLE
Tweak npq participant endpoints api docs

### PR DESCRIPTION
**Pull request by leandroalemao**
### Context

- Ticket: https://dfedigital.atlassian.net/browse/CPDLP-3082

During PO testing the ticket above, we found some discrepancies between the npq participant serializer and the api docs.

### Changes proposed in this pull request

Fix those discrepancies in api v1/v2/v3 docs for npq participant endpoints.

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@
   # confirmation, reset password and unlock tokens in the database.
   # Devise will use the `secret_key_base` as its `secret_key`
   # by default. You can change it below and use your own secret key.
-  config.secret_key = Rails.application.secrets.secret_key_base
+  config.secret_key = Rails.application.credentials.secret_key_base
```
Comment (cwrw) [inline config/initializers/devise.rb:17]: discussed in person: could we separate this pr into it's own pr and check our credentials have those set correctly

File: spec/models/induction_record_spec.rb [hunk 2]
```diff
@@ -57,7 +57,7 @@
       end
 
       it "includes completed induction, current, transferring, transferred, training status withdrawn and training status deferred" do
-        expect(described_class.school_dashboard_relevant).to eq(expected_included)
+        expect(described_class.school_dashboard_relevant).to match_array(expected_included)
```
Comment (cwrw) [inline spec/models/induction_record_spec.rb:60]: discussed in person: does order matter here? if so should we fix the test?

**Review by cwrw — COMMENTED** [review]
just the comments we discussed in person, rest looks great, thanks for fixing those @leandroalemao !

File: config/initializers/devise.rb (hunk 1 above)
Comment (leandroalemao) [reply config/initializers/devise.rb:17]: Moving this to another PR.. 
Checked all credentials and all have `secret_key_base` set correctly.

**Review by cwrw — APPROVED** [review]
